inner.html

```diff
@@ -1,173 +1,223 @@
-                    <p class="price">10</p>
-                </div>
-            </li>
-            <li class="items-container__item">
-                <div class="photo">
-                    <img class="item" src="https://placeimg.com/640/480/any/2" alt="Item">
-                </div>
-                <div class="information">
-                    <p class="description">Lorem ipsum dolor si!</p>
-                    <p class="price">10</p>
-                </div>
-            </li>
-            <li class="items-container__item">
-                <div class="photo">
-                    <img class="item" src="https://placeimg.com/640/480/any/1" alt="Item">
-                </div>
-                <div class="information">
-                    <p class="description">Lorem ipsum dolor si!</p>
-                    <p class="price">10</p>
-                </div>
-            </li>
-            <li class="items-container__item">
-                <div class="photo">
-                    <img class="item" src="https://placeimg.com/640/480/any/11" alt="Item">
-                </div>
-                <div class="information">
-                    <p class="description">Lorem ipsum dolor si!</p>
-                    <p class="price">10</p>
-                </div>
-            </li>
-            <li class="items-container__item">
-                <div class="photo">
-                    <img class="item" src="https://placeimg.com/640/480/any/12" alt="Item">
-                </div>
-                <div class="information">
-                    <p class="description">Lorem ipsum dolor si!</p>
-                    <p class="price">10</p>
-                </div>
-            </li>
-            <li class="items-container__item">
-                <div class="photo">
-                    <img class="item" src="https://placeimg.com/640/480/any/13" alt="Item">
-                </div>
-                <div class="information">
-                    <p class="description">Lorem ipsum dolor si!</p>
-                    <p class="price">10</p>
-                </div>
-            </li>
-        </ul>
-    </section>
-  </main>
-  <footer class="footer-container">
-    <div class="footer__item">
-        <a href="#">About us</a>
-    </div>
-    <div class="footer__item">
-        <a href="#">Address</a>
-    </div>
-    <div class="footer__item">
-        <a href="#">Follow us</a>
-    </div>
-</footer>
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Task</title>
+		<link rel="preconnect" href="https://fonts.gstatic.com" />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&family=Ubuntu:wght@400;700&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="styles.css" />
+	</head>
+	<body>
+		<header>
```

Comment:
Nitpick: Do you find this formatting more readable?  
Personally I prefer 2 spaces in html code. 😃